tests/framework/framework.go

```diff
@@ -24,14 +24,15 @@ import (
 )
 
 const (
-	NsCreateTime = 30 * time.Second
-	NsDeleteTime = 5 * time.Minute
+	NsCreateTime   = 30 * time.Second
+	NsDeleteTime   = 5 * time.Minute
+	DefaultTimeout = 30 * time.Second
```

Comment:
again, pls use a more descriptive const name.

Reply:
So this DefaultTimeout is a generic timeout value in the case you don't have a more specific const defined. I won't want to end up with 50 XyTimeout, one for each type of entity. If we do want specific timeouts for a particular entity because of something in that entity makes it necessary to have different defaults, I would suggest we put it in the const block for that entity utility file, instead of in the framework const block.

This particular one, is generic, and the Ns ones are in here because they are part of the framework file.